classes/ActionScheduler_AdminView.php

```diff
@@ -112,6 +113,94 @@ protected function get_list_table() {
 		return $this->list_table;
 	}
 
+	/**
+	 * Action: admin_notices
+	 *
+	 * Maybe check past-due actions, and print notice.
+	 *
+	 * @uses $this->check_pastdue_actions()
+	 */
+	public function maybe_check_pastdue_actions() {
+
+		# Filter to prevent checking actions (ex: inappropriate user).
+		if ( ! apply_filters( 'action_scheduler_check_pastdue_actions', true ) )
```

Comment:
Sorry, I just noticed. This should have a `current_user_can( 'manage_options` )` as only admins can access the SA screen.

Reply:
Resolved in a17cf37.